lib/videoroom_web/router.ex

```diff
@@ -9,6 +9,7 @@ defmodule VideoroomWeb.Router do
     pipe_through :api
 
     get "/room/:room_name", RoomController, :show
+    get "/room/:room_name/exist", RoomController, :room_exist
```

Comment:
**thought (non-blocking):** This feels like bad API design, but I don't have neither a suggestion nor feel strong enough about it

Reply:
Yup, it is, but I don't think it is worth fixing the API design if we plan to migrate this backend to RoomManager in the near future.